clang/test/dpct/helper_files_ref/include/device.hpp

```diff
@@ -640,6 +640,44 @@ static inline unsigned int get_device_id(const sycl::device &dev){
   return dev_mgr::instance().get_device_id(dev);
 }
 
+inline void
+has_capability_or_fail(const sycl::device &dev,
+                       const std::initializer_list<sycl::aspect> &props) {
+  for (const auto &it : props) {
+    if (dev.has(it))
+      continue;
+    switch (it) {
+    case sycl::aspect::fp64:
+      throw std::runtime_error("'double' is not supported in this device");
```

Comment:
how about print device name of this SYCL device,  instead of "this device"? (friendly for user)

Reply:
fixed